src/platform/backends/qemu/CMakeLists.txt

```diff
@@ -24,3 +24,7 @@ target_link_libraries(qemu_backend
   ip_address_pool
   ssh
   yaml)
+
+install(PROGRAMS check_kvm_support
+  DESTINATION bin
+  COMPONENT multipassd)
```

Comment:
Does this change anything?

Reply:
Well, "COMPONENT" was specified for install of multipassd in src/daemon/CMakeLists.txt, so I think we should specify the same component for check_kvm_support since is should be installed when multipassd is.

Reply:
Ack